modules/ROOT/pages/constraints/examples.adoc

```diff
@@ -1245,6 +1257,78 @@ Constraint already exists: Constraint( id=23, name='seriesOrder', type='NODE PRO
 ======
 
 
+[discrete]
+[[constraints-create-node-property-type-constraints-on-invalid-types]]
+=== Creating constraints on invalid types will fail
+
+
+.+CREATE CONSTRAINT+
+======
+
+Create a node property type constraint restricting the property `imdbScore` to map values on nodes with the `Movie` label:
+
+.Query
+[source, cypher, role=test-fail]
+----
+CREATE CONSTRAINT score
+FOR (movie:Movie) REQUIRE movie.imdbScore IS :: MAP
+----
+
+In this case, the constraint cannot be created because the `MAP` type is not allowed in property type constraints.
```

Comment:
I took the `In this case, ` part from existing failing examples so it might be nice to keep that for consistency?
```suggestion
In this case, the constraint cannot be created because values of type `MAP` cannot be stored in properties and therefore are not permitted in property type constraints.
```

Reply:
Sure, we can keep it. I removed it because it seems unnecessary.